autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/ExpandedCourseCard.tsx

```diff
@@ -48,15 +51,21 @@ const ExpandedCourseCard: React.FC<ExpandedCourseCardProps> = ({ onCollapse, id
   return (
     <Card>
       <div className={styles.header}>
-        <div className={styles.headerGroup}>
+        <div
+          className={styles.headerGroup}
+          onClick={(): void => onRemove(id)}
+          role="button"
+          tabIndex={0}
+          onKeyPress={(): void => onRemove(id)}
+        >
           <RemoveIcon />
           Remove
         </div>
         <div
           className={styles.headerGroup}
           onClick={(): void => onCollapse(course)}
           role="button"
-          tabIndex={0}
+          tabIndex={-1}
```

Comment:
If the remove button is tab index 0, shouldn't the collapse button be tab index +1?

Reply:
While that makes sense to me, whenever I tried to make it a positive value I got the eslint error: `Avoid positive integer values for tabIndex.eslint(jsx-a11y/tabindex-no-positive)` (documentation for the rule [here](https://github.com/reactjs/react-a11y/blob/master/docs/rules/tabindex-no-positive.md))

Turns out that they should both be zero 0? That's the only way both Remove and Collapse can be both be selected with tab for me.